.github/workflows/deploy-ALL-connectors-1.yaml

```
@@ -0,0 +1,154 @@
+---
+# This workflow will help deploy ALL
+
+name: Deploy ALL connectors 1
+
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - '*'
+  release:
+    types: [released]
+
+jobs:
+  base-check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Wait for run-manifest in deploy ALL to finish
+        uses: lewagon/wait-on-check-action@v1.1.1
+        with:
+          ref: ${{ github.ref }}
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          check-regexp: (run-manifest|run-dbconfigs)
```

Can likely be shortened to
```suggestion
          check-regexp: run-(manifest|dbconfigs)
```

---

shorter maybe but I think as it currently is more readable